gluten-core/src/main/scala/org/apache/gluten/GlutenPlugin.scala

```diff
@@ -143,6 +143,11 @@ private[gluten] class GlutenDriverPlugin extends DriverPlugin with Logging {
     }
     conf.set(SPARK_SESSION_EXTS_KEY, extensions)
 
+    // sql adaptive customCostEvaluatorClass
+    conf.set(
+      SQLConf.ADAPTIVE_CUSTOM_COST_EVALUATOR_CLASS.key,
+      "org.apache.spark.sql.execution.adaptive.GlutenCostEvaluator")
+
```

Comment:
Let's have a check to raise an error when there was already another cost evaluator set by user.